cmd/route-emitter/main_test.go

```diff
@@ -135,10 +135,28 @@ var _ = Describe("Route Emitter", func() {
 						PrivateInstanceId: instanceKey.InstanceGuid,
 					})))
 				})
+
+				Context("and the instance has a route with a route service url", func() {
+					BeforeEach(func() {
+						routes := newRoutes(hostnames, containerPort, "https://awesome.com")
+						desiredLRP.Routes = routes
+					})
+
+					It("emits routes immediately including the route service url", func() {
```

Comment:
Instead of adding multiple JustBeforeEach's and adding this new context, it would be better to just add the route service url to line 62 and update the existing "emits its routes immediately" test.